src/ButtonSelect/ButtonSelect.js

```diff
@@ -0,0 +1,46 @@
+
+import React, { useState, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button';
+
+const menuStyles = {
+  width: '100%',
+  top: '100%'
+};
+
+const ButtonSelect = ({
+  text,
+  children,
+  ...Attr
+}) => {
+  const [toggled, setToggled] = useState(false);
+
+  return (
+    <Fragment>
```

Comment:
Yo don't need this Fragment, Fragment is just necessary when you have more than one jsx node.